### ISSUE TITLE
Account Creation - Upstream request timeout 

### ISSUE THREAD

**Issue by keileyo** [cfpb/hmda-platform#2317]
I am consistently getting an "upstream request timeout" when creating a new account. The issue is resolved by refreshing the page, but our users wouldn't know to do that. 

We should:
1) resolve the issue
2) if possible, update the text received when the error happens including instructions to refresh the page (this may belong in UI).

**Comment by omnipresent07** [comment]
Just confirming - When the "upstream request timeout" error happens, is the institution checkbox selected? I opened this issue https://github.com/cfpb/hmda-platform/issues/2303 which was also getting the same error but only when the checkbox was not selected.

**Comment by keileyo** [comment]
yeah, in this case i am using my @cfpb.gov email domain and saw the error when i had selected both bank 0 and bank 1.

**Comment by awolfe76** [comment]
Are you using IE 11 for this? Does it happen in all browsers?

**Comment by awolfe76** [comment]
This does happen in all browsers.

**Comment by zencircle** [comment]
@awolfe76 curious if need to define` timeout_ms`value in mapping configuration https://www.getambassador.io/reference/mappings/#mapping-configuration

**Comment by zencircle** [comment]
Proposed change
kubectl edit service keycloak-ambassador

apiVersion: v1
kind: Service
metadata:
  annotations:
    getambassador.io/config: |
      apiVersion: ambassador/v0
      kind: Mapping
      name: keycloak_https_mapping
      prefix: /auth/
      rewrite: /auth/
      service: keycloak-http
      timeout_ms: 4000  <---- Add

**Comment by zencircle** [comment]
Marathon-lb timeout for  keycloak 

backend hmda_keycloak_10112  
  ...
  option  httpchk GET /  
  timeout check 20s

**Comment by zencircle** [comment]
On 12/13 - Updated timeout configuration to `5s`
